wordpress-mobile/WordPress-iOS
# add base localization for the intents extension

Fixes #NA

To test:

It's not really a test, we will be able check that the localization issue on the `intents` extension is resolved once we re-publish to the App Store

PR submission checklist:
